core/cog-webkit-utils.c

```diff
@@ -98,22 +98,19 @@ format_tls_error (GTlsCertificateFlags errors)
     return g_string_free (str, FALSE);
 }
 
-
 gboolean
-cog_handle_web_view_load_failed_with_tls_errors (WebKitWebView       *web_view,
-                                                 char                *failing_uri,
-                                                 GTlsCertificate     *certificate,
-                                                 GTlsCertificateFlags errors,
-                                                 void                *user_data)
+cog_web_view_load_error_page (WebKitWebView *web_view,
+                              char *failing_uri,
+                              GTlsCertificateFlags errors)
```

Comment:
I would keep the `GTlsCertificate` parameter for future-proofing. We may want to improve the built-in error pages at some point to include some information derived from the certificate.